testing-guidance/workflow-tests/scripting/script-python-tests.md

```diff
@@ -0,0 +1,37 @@
+1. Asset processor errors when importing/executing python scripts
+2. Warnings or errors that appear in the Editor console
+3. Unresponsive UI elements 
+
+### Platforms:
+- Windows
+- Linux
+
+### Documents and Common Terms
+[O3DE Documentation: Editor Python Bindings Gem](https://www.o3de.org/docs/user-guide/gems/reference/script/python/editor-python-bindings/) <br>
+
+
+<b>Python Scripts</b>: Tool that lists python script samples and allows to execute those scripts.
+
+<b></b>
+## Area: Opening Python Scripts tool and executing sample scripts
+
+### Project Requirements
+Any project can be used, the Editor Python Bindings gem and its gem dependencies are required.
+<br/>White box Gem is required to visualize easily the script execution as it includes a python scripts that create various entities with meshes.
+
+**Product:** 
+Visible and responsive Python Script tool.
+
+**Suggested Time Box:** 
+15 minutes
+
+| Workflow                                                            | Requests                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Things to Watch For                                                                                                                                                                                                                               |
+|---------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|Executing python script samples| * Open Python Scripts tool via Tools -> Other -> Python Scripts <br/> * Select a script to execute (e. g. Sphere, Staircase)![Python Scripts Tool](images/Python_Scripts_Tool.png)<br/>* Execute selected script using dedicated Execution button.| * Script executes without errors/warnings in the console. <br/> * Entity with mesh component is created and visible in Viewport.
```

Comment:
Unfortunately, lists inside of an MD table must use HTML list formatting.

Reply:
Applied HTML list formatting.